src/main/java/org/apache/mesos/hdfs/util/HDFSConstants.java

```diff
@@ -22,4 +22,8 @@
   public static final String NODE_EXECUTOR_ID = "NodeExecutor";
   public static final String NAME_NODE_EXECUTOR_ID = "NameNodeExecutor";
 
+  // Path to Store HDFS Binary
+  public static final String MESOSPHERE_OPT_PATH = "/opt/mesosphere/";
```

Comment:
/opt/mesosphere will be the generic path for different binaries (HDFS, Cassandra, etc). I think this should be defined and not configurable. Most likely this should be defined in Mesos itself, but I left it as a constant. I will rename it MESOSPHERE_FRAMEWORK_MNT_PATH.